commands/customObject/create.ts

```diff
@@ -50,16 +65,22 @@ exports.handler = async options => {
       })
     );
   }
-};
+}
+
+export function builder(yargs: Argv): Argv<CustomObjectCreateArgs> {
+  addConfigOptions(yargs);
+  addAccountOptions(yargs);
+  addUseEnvironmentOptions(yargs);
+
+  yargs.positional('name', {
+    describe: i18n(`${i18nKey}.positionals.name.describe`),
+    type: 'string',
+  });
+
+  yargs.option('path', {
+    describe: i18n(`${i18nKey}.options.path.describe`),
+    type: 'string',
+  });
```

Comment:
Nit: Do we need to split these or can we keep them chained?

Reply:
You can make the mock return itself.  Here's [an example](https://github.com/HubSpot/hubspot-cli/blob/main/commands/secret/__tests__/deleteSecret.test.ts#L16-L24) of how you did it in an earlier test